cpp/src/ice2slice/Gen.cpp

```diff
@@ -561,8 +561,6 @@ Gen::TypesVisitor::visitInterfaceDefStart(const InterfaceDefPtr& p)
         }
 
         ExceptionList throws = op->throws();
-        throws.sort();
-        throws.unique();
```

Comment:
We no longer sort our exception specifications, because there's no point to calling `unique`.
The Slice compiler already ensures this for us.